### PR TITLE
Added article for 0.19.0 release

### PR DESCRIPTION
**Pull request by jasonbcox**

### CODE REVIEW

File: _posts/2019-02-20-bitcoin-abc-0.19.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The goal of Bitcoin ABC is to help create sound money that is usable globally in order to drive freedom and prosperity for everyone. This [vision](/2018-08-24-bitcoin-abc-vision) has not changed.
+
+Keeping in line with the [Bitcoin Cash roadmap](https://www.bitcoincash.org/roadmap.html), the Bitcoin ABC team is proud to present the May 2019 protocol upgrade release. Bitcoin ABC 0.19.0 is now available and can be downloaded at: [https://download.bitcoinabc.org/0.19.0](https://download.bitcoinabc.org/0.19.0/)
+
+The features going into every protocol upgrade are planned, specified [1], and developed far in advance in order to give users, businesses, miners, and all other ecosystem participants ample time to prepare. In order to assist technical preparations for the upgrade, a testnet is available where the upgrade features can be trialed and tested. Businesses and services built on the Bitcoin Cash network can use this testnet to ensure compatibility, or to get started building products to make use of the upgrade features. Testnet information is available [here](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/workgroups/wg-testing/2019-05-15_upgrade_testnet.md).
```
Comment (Mengerian) [inline _posts/2019-02-20-bitcoin-abc-0.19.0.md]: I'd add the link to the specification directly here

File: _posts/2019-02-20-bitcoin-abc-0.19.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Similar to the last upgrade, the Bitcoin ABC 0.19.0 release has been made available three months in advance of the upgrade activation, which is scheduled to occur on May 15th, 2019. The protocol upgrade, once activated, will enable the following features:
+* **Schnorr signatures** [2] for enhanced privacy and performance, for both single- and multiple-signature transactions.
```
Comment (Mengerian) [inline _posts/2019-02-20-bitcoin-abc-0.19.0.md]: Add link here also

File: _posts/2019-02-20-bitcoin-abc-0.19.0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* **Segwit recovery transactions** [3].  With the last protocol upgrade, the enforcement of the CLEANSTACK rule made it impossible to recover funds from Segwit transactions.  With the upgrade, there will be an exception to this rule to enable recovery of these funds once again.
```
Comment (Mengerian) [inline _posts/2019-02-20-bitcoin-abc-0.19.0.md]: And link here also

File: _posts/2019-02-20-bitcoin-abc-0.19.0.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Prepare for the protocol upgrade today with 0.19.0
```
Comment (Mengerian) [inline _posts/2019-02-20-bitcoin-abc-0.19.0.md]: Maybe should expand to:
"Prepare for the Bitcoin Cash protocol upgrade today with Bitcoin ABC 0.19.0"

File: _posts/2019-02-20-bitcoin-abc-0.19.0.md (hunk 1 above)
Comment (jasonbcox) [reply _posts/2019-02-20-bitcoin-abc-0.19.0.md]: Good ideas about the direct links

**Pull request merged**